File: C/Assignment-2/Prog11.c
#include<stdio.h>

int isPrime(int num)
{
    for(int i = 2; i <= num/2; i++){
        if (num%i==0)
            return 0;
    }
    return 1;
}
int main(){
    int number = 0;
    printf("Enter a number: ");
    scanf("%d", &number);
    
    for(int i = 2; i <= number; i++){
        if (isPrime(i))
            printf("%d, ", i);
    }
    return 0;
}

File: Python/Assignment-2/Prog11.py
def isPrime(num:int) -> bool:
    for i in range(2,num//2 + 1):
        if num%i==0:
            return False
    return True

upper, lower = map(int, input("Enter the upper and the lower range : ").split())
primes = []
for i in range(lower, upper + 1):
    if isPrime(i):
        primes.append(i)

print(primes)